src/plugins/credential_management/public/components/create_credential_wizard/create_credential_wizard.tsx

```diff
@@ -159,8 +163,12 @@ export class CreateCredentialWizard extends React.Component<
 
   render() {
     const content = this.renderContent();
-
-    return (
+    const { maskLoading } = this.state;
+    return maskLoading ? (
+      <EuiOverlayMask>
+        <EuiLoadingSpinner size="xl" />
+      </EuiOverlayMask>
+    ) : (
```

Comment:
Can we decouple this as an individual component since it's reused multiple place in code?